ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -2806,30 +2793,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-gcp-ipi-proxy
-- as: gcp-ipi-realtime-kernel-fips-p2-f14
-  cron: 11 8 13,28 * *
+- as: gcp-ipi-rt-fips-p2-f14
```

Comment:
this one is already turned into `gcp-ipi-workload-identity-rt-fips-p1-f14`, right?

Reply:
It turned into `gcp-ipi-workload-identity-rt-fips-p2-f14` in 4.13.